frontend/src/scenes/UpgradeModal.tsx

```diff
@@ -1,28 +1,41 @@
-import Modal from 'antd/lib/modal/Modal'
+import { LemonButton, LemonModal } from '@posthog/lemon-ui'
 import { useActions, useValues } from 'kea'
 import { capitalizeFirstLetter } from 'lib/utils'
+import { posthog } from 'posthog-js'
 import { sceneLogic } from './sceneLogic'
+import { urls } from './urls'
 
 export function UpgradeModal(): JSX.Element {
     const { upgradeModalFeatureNameAndCaption } = useValues(sceneLogic)
-    const { hideUpgradeModal, takeToPricing } = useActions(sceneLogic)
+    const { hideUpgradeModal } = useActions(sceneLogic)
 
     const [featureName, featureCaption] = upgradeModalFeatureNameAndCaption ?? []
 
     return (
-        <Modal
-            title="Unleash PostHog's Full Power"
-            okText="Upgrade Now"
-            cancelText="Maybe Later"
-            onOk={takeToPricing}
-            onCancel={hideUpgradeModal}
-            visible={!!featureName}
+        <LemonModal
+            title="Unleash PostHog's full power"
+            footer={
+                <>
+                    <LemonButton type="secondary" onClick={hideUpgradeModal}>
+                        Maybe later
+                    </LemonButton>
+                    <LemonButton
+                        type="primary"
+                        to={urls.organizationBilling()}
```

Comment:
Unfortunately I'm not fully up to speed with our billing yet. Is this modal only shown for the cloud version? Before this change we would adapt the url based on wether this is running in cloud or not.

Reply:
This is shown on self-hosted as well. The change to the URL is definitely intentional though – thanks to recent work by Ben & Em, it's finally possible to upgrade PostHog without leaving the app. [Checked this with Em on Slack to be sure.](https://posthog.slack.com/archives/C047K2ETZ3R/p1671463234431379)

Reply:
Awesome then!